src/definitions.ts

```diff
@@ -260,7 +260,7 @@ export interface AgreementSizeExceededPayload {
 
 export interface MultiaddrAnnouncementPayload {
   agreementReference: string
-  multiaddr: string
+  peerId: string
```

Comment:
Can we change payload here from multiadress to peerId?

Reply:
Sure, sounds good!